### PR TITLE
(MODULES-9333) Updates to unit/acceptance tests

### PR DESCRIPTION
**Pull request by eimlav**
* Enable rspec-mocks

* Use serverspec for file assertions

* Use rspec-expectations

### CODE REVIEW

File: spec/acceptance/basic_functionality/puppet_apply_dsc_manifest_spec.rb [hunk 1]
```diff
@@ -1,72 +1,46 @@
 require 'spec_helper_acceptance'
-
-describe 'Puppet apply tests' do
+require 'pry'
```
Comment (carabasdaniel) [inline spec/acceptance/basic_functionality/puppet_apply_dsc_manifest_spec.rb]: Do you need pry here ?

Comment (eimlav) [reply spec/acceptance/basic_functionality/puppet_apply_dsc_manifest_spec.rb]: updated.

**Comment by michaeltlombardi** [comment]
@eimlav I'm happy to merge this one if these acceptance tests have been run somewhere - perhaps in an adhoc run or locally?

**Comment by eimlav** [comment]
@michaeltlombardi Tests ran successfully in adhoc.

Screen Shot 2019-07-09 at 10 31 32

**Pull request merged**